src/main/java/htsjdk/samtools/SAMFileWriter.java

```diff
@@ -43,6 +43,11 @@
 	 */
 	void setProgressLogger(final ProgressLoggerInterface progress);
 
+	/** If true writers that are writing pre-sorted records should check the order during writing. */
```

Comment:
Should this throw unsupported instead by default?

Reply:
Done.